src/Compilers/CSharp/Portable/Binder/ExecutableCodeBinder.cs

```diff
@@ -161,7 +161,7 @@ public void ValidateIteratorMethods(DiagnosticBag diagnostics)
                     diagnostics.Add(ErrorCode.ERR_VarargsIterator, iterator.Locations[0]);
                 }
 
-                if (((iterator as SourceMemberMethodSymbol)?.IsUnsafe == true || (iterator as LocalFunctionSymbol)?.IsUnsafe == true)
+                if ((iterator is SourceMemberMethodSymbol { IsUnsafe: true } || (iterator as LocalFunctionSymbol)?.IsUnsafe == true)
```

Comment:
curious why the other branch didn't get updated.

Reply:
Only one instance is applied per file by the refactoring runner.